projects/plugins/jetpack/modules/masterbar/admin-menu/class-jetpack-admin-menu.php

```diff
@@ -231,7 +233,11 @@ public function add_users_menu() {
 	 */
 	public function add_tools_menu() {
 		add_menu_page( esc_attr__( 'Tools', 'jetpack' ), __( 'Tools', 'jetpack' ), 'publish_posts', 'tools.php', null, 'dashicons-admin-tools', 75 );
-
+		// TODO: remove locale restriction when blaze is open to everyone.
+		$user_locale = ( new Manager() )->get_connected_user_data()['user_locale'];
+		if ( 'en' == $user_locale ) {
```

Comment:
Maybe we could do it like that instead, to handle both en and en-gb, like on WordPress.com?

```suggestion
		$user_data = ( new Manager() )->get_connected_user_data();
		if (
			! empty( $user_data['user_locale'] )
			&& in_array( $user_data['user_locale'], array( 'en', 'en-gb' ), true )
		) {
```

Reply:
>  You would have to delete the transient starting with jetpack_connected_user_data_, and then visit the Jetpack dashboard in wp-admin to refresh the data, and that should work.

Yes, it works. Learned about transients today 🙏 
I've also updated the unit tests so they're all good now!